src/main/resources/static/app/request/edit-request.html

```diff
@@ -871,9 +871,9 @@ <h4 class="h4 h4-palga">{{'Final checks before data delivery'|translate}}</h4>
                 </div> <!--end modal-body-->
 
                 <div class="modal-footer">
-                    <a id="cancel-new-request" class="btn btn-default pull-left req-edit-button" ng-click="cancel(request);">
+                    <button id="cancel-new-request" class="btn btn-default pull-left req-edit-button" ng-click="cancel(request);">
```

Comment:
I've added tabindex="-1" to the button which means it can no longer be tabbed too.